src/static/css/index.css

```diff
@@ -17,6 +17,19 @@ header.alt-bg {
   background-color: transparent;
 }
 
+header .container {
+  max-width: none;
```

Comment:
Why no max-width? I agree that is the current design but seems weird that Home page is very wide, but rest are constrained. Maybe not part of this PR and maybe one to raise in an issue and discuss, but looks weird to me none-the-less so thought I'd ask.

Reply:
Sure, that's a good question to ask.

I didn't ask about the current design decisions either, as I wasn't involved from the beginning and that wouldn't have been fair coming from me. 

It would be a good idea to raise this as part of a different issue and besides that, we should take into account the fact that the header "jumps" on the index due to the "Start exploring" right upper button. I'm pretty sure that wasn't on purpose and we should address that too maybe.

Reply:
Raised #574 . Let's resolve this for this PR.